### PR TITLE
BZ1683094 imagestream tag updates

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1683094

### CODE REVIEW

File: modules/images-imagestream-external-image-tags.adoc [hunk 1]
```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
 
 * Add tags pointing to internal or external images, by using the `oc tag` command
  for all tag-related operations:
- +
++
 ----
-oc tag <repositiory/image> <image-name:tag>
+oc tag <image-name:tag> <image-name:tag>
```
Comment (xiuwang) [inline modules/images-imagestream-external-image-tags.adoc]: The cmd here 'oc tag <repositiory/image> <image-name:tag>'  is correct, the purpose is tagging an external image.

File: modules/images-imagestream-external-image-tags.adoc [hunk 2]
```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
 
 * Add tags pointing to internal or external images, by using the `oc tag` command
  for all tag-related operations:
- +
++
```
Comment (xiuwang) [inline modules/images-imagestream-external-image-tags.adoc]: There are other three '+' strings in this doc, need removed

**Comment by xiuwang** [comment]
My suggest: change cmd "oc tag <image-name:tag> <image-name:tag> " to  "oc tag <image-name:tag1> <image-name:tag2>" in the doc
https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/images-imagestream-adding-tags.adoc

File: modules/images-imagestream-external-image-tags.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/images-imagestream-external-image-tags.adoc]: There was a spacing issue previously that caused the '+'s to build. I removed the space and it looks like it's building correctly to align the example context with the procedure.

File: modules/images-imagestream-external-image-tags.adoc [hunk 3]
```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
...
 ----
-oc tag <repositiory/image> <image-name:tag>
```
Comment (xiuwang) [inline modules/images-imagestream-external-image-tags.adoc]: The cmd here is no need to change, it's correct.

File: modules/images-imagestream-external-image-tags.adoc [hunk 4]
```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
...
+oc tag <image-name:tag1> <image-name:tag2>
 ----
```
Comment (xiuwang) [inline modules/images-imagestream-external-image-tags.adoc]: My suggest: change cmd "oc tag image-name:tag image-name:tag " to "oc tag image-name:tag1 image-name:tag2" in the doc
https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/images-imagestream-adding-tags.adoc

Comment (bmcelvee) [reply modules/images-imagestream-external-image-tags.adoc]: Is the most recent correct?

----
oc tag <repository/image> <image-name:tag>
oc tag <image-name:tag1> <image-name:tag2>
----

or  should it be:

----
oc tag <repository/image-name:tag1> <image-name:tag2>
----

Thanks again!

Comment (xiuwang) [reply modules/images-imagestream-external-image-tags.adoc]: "oc tag <repository/image> <image-name:tag>" is correct here, no need change.

I suggest to modify  cmd "oc tag image-name:tag image-name:tag " to "oc tag image-name:tag1 image-name:tag2" in  another doc https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/images-imagestream-adding-tags.adoc

Comment (bmcelvee) [reply modules/images-imagestream-external-image-tags.adoc]: Thank you!

**Comment by xiuwang** [comment]
LGTM
Brandi, thanks~

**Pull request merged**